go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/viper v1.12.0
 	github.com/stretchr/testify v1.8.0
-	github.com/tektoncd/pipeline v0.35.0
+	github.com/tektoncd/pipeline v0.37.0
```

Comment:
This dependency had to be updated due to changes in API on the server side, which are incompatible with an old client.